src/Sfa.Das.ApprenticeshipInfoService.Core/Helpers/ActiveApprenticeshipChecker.cs

```diff
@@ -7,45 +7,14 @@ namespace Sfa.Das.ApprenticeshipInfoService.Core.Helpers
 {
     public class ActiveApprenticeshipChecker : IActiveApprenticeshipChecker
     {
-        private readonly IConfigurationSettings _configurationSettings;
-
-        public ActiveApprenticeshipChecker(IConfigurationSettings configurationSettings)
-        {
-            _configurationSettings = configurationSettings;
-        }
-
-        public bool CheckActiveFramework(string frameworkId, DateTime? effectiveFrom, DateTime? effectiveTo)
-        {
-            return DateHelper.CheckEffectiveDates(effectiveFrom, effectiveTo) || IsSpecialLapsedFramework(frameworkId);
-        }
-
-        public bool CheckActiveStandard(string standardId, DateTime? effectiveFrom, DateTime? effectiveTo)
-        {
-            return DateHelper.CheckEffectiveDates(effectiveFrom, effectiveTo) || IsSpecialLapsedStandard(standardId);
-        }
-
-        private bool IsSpecialLapsedFramework(string frameworkId)
-        {
-            var lapsedFrameworks = _configurationSettings.FrameworksExpiredRequired;
-
-            return CheckValueIntoList(frameworkId, lapsedFrameworks);
-        }
-
-        private bool IsSpecialLapsedStandard(string standardId)
+        public bool CheckActiveFramework(DateTime? effectiveFrom, DateTime? effectiveTo)
         {
-            var lapsedStandards = _configurationSettings.StandardsExpiredRequired;
-
-            return CheckValueIntoList(standardId, lapsedStandards);
+            return DateHelper.CheckEffectiveDates(effectiveFrom, effectiveTo);
         }
 
-        private static bool CheckValueIntoList(string apprenticeshipId, List<string> lapsedApprenticeships)
+        public bool CheckActiveStandard(DateTime? effectiveFrom, DateTime? effectiveTo)
```

Comment:
Can this be renamed to IsActiveStandard? More conclusive as to what its doing then 😄